### ISSUE TITLE
Problems running locally on Windows 10

### ISSUE THREAD

**Issue by Coder2012** [Mugen87/yuka#8]
Following your instructions:

```
npm install
npm start
```

The output in the console is:

```
> npm run dev


> yuka@0.0.1 dev C:\git\yuka
> concurrently "rollup -c "config/rollup/config.js" -w" "http-server -c-1"

[0] rollup v1.12.0
[0] bundles src/yuka.js → build\yuka.js, build\yuka.module.js...
[1] Starting up http-server, serving ./
[1] Available on:
[1]   http://192.168.1.125:8080
[1]   http://127.0.0.1:8080
[1] Hit CTRL-C to stop the server
[0] created build\yuka.js, build\yuka.module.js in 1s
```

After opening either of the available urls Chrome says:

This page isn’t working 127.0.0.1 sent an invalid response.
ERR_INVALID_REDIRECT

Nothing is rendered at all.

**Comment by Mugen87** [comment]
@robp94 Do you have the same issue on your Windows machine?

**Comment by Mugen87** [comment]
Probably related: https://github.com/http-party/http-server/issues/525

@Coder2012 Can you please check if a downgrade to `0.9.0` of `http-server` solves the issue for your? Just edit the following line an perform a new install.

https://github.com/Mugen87/yuka/blob/774d48c91cdb1fc2c3263abe4fcfd9dd4938868a/package.json#L52

**Comment by Coder2012** [comment]
@Mugen87 Yes downgrading `http-server` works, this does however have a dependency on a vulnerable version of `ecstatic`.

This is ok for me personally as I'm just browsing your examples locally.

**Comment by Mugen87** [comment]
Good to know it's related to this issue. I'm curious what @robp94 reports. If he has the same issue, the downgrade seems okay to me since it's just a local dev server.

**Comment by robp94** [comment]
> @robp94 Do you have the same issue on your Windows machine?

Yes I get the same error.

**Comment by Mugen87** [comment]
Thanks for testing! In this case let's downgrade `http-server` to `0.9.0`. We will get some additional security warnings however the risk is tolerable since we are using the package just for a local dev server.

Fix: https://github.com/Mugen87/yuka/commit/c165b8aa6f72097c62fab74e082439b17c2fa299

**Comment by Mugen87** [comment]
FYI: `http-server` has been removed now. The project uses [serve](https://www.npmjs.com/package/serve) now.

**Comment by thornjad** [comment]
For what it's worth, this issue is fixed in `http-server@0.12.0`
